#!/usr/bin/python3
"""Checks for errors unlike previous tasks"""
from urllib import request, error
from sys import argv


if __name__ == "__main__":
    